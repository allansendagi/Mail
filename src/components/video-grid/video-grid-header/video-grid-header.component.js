import React from 'react';
import './video-grid-header.styles.scss';

export function VideoGridHeader(props) {
  return (
    <div className='video-grid-header'>
      <span className='title'>{props.title}</span>
    </div>
  );
}