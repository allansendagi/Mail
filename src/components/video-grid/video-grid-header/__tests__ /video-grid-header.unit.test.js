import React from 'react';
import {shallow, configure} from 'enzyme';
import {VideoGridHeader} from '../video-grid-header.component';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});
describe('VideoGridHeader', () => {
  test('renders without props', () => {
    const wrapper = shallow(<VideoGridHeader/>);
    expect(wrapper).toMatchSnapshot();
  });
  test('renders with empty string header', () => {
    const wrapper = shallow(<VideoGridHeader title=''/>);
    expect(wrapper).toMatchSnapshot();
  });
  test('renders with title', () => {
    const wrapper = shallow(<VideoGridHeader title='Autos & Vehicles'/>);
    expect(wrapper).toMatchSnapshot();
  });
});
