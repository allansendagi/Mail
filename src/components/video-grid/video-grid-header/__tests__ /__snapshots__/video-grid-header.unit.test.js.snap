// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoGridHeader renders with empty string header 1`] = `ShallowWrapper {}`;

exports[`VideoGridHeader renders with title 1`] = `ShallowWrapper {}`;

exports[`VideoGridHeader renders without props 1`] = `ShallowWrapper {}`;
